### DISCUSSION TITLE
[视频]Bad Apple!! feat.nomico

### DISCUSSION THREAD

**Discussion by muxi0330** [share121/inter-knot#835] [其他]
![Bad_Apple](https://github.com/user-attachments/assets/3a50492b-e23d-41e0-93cc-deb3f2d2d824)

https://github.com/user-attachments/assets/c86c86ba-a8b1-477e-80f8-587f08f2c97c

**Comment by annexen** [comment]
woc,车万

**Comment by fy-sec** [comment]
经典老番

**Comment by Y-0930** [comment]
经典

**Comment by Y-0930** [comment]
为啥我声音开不开啊

**Comment by lizard315315** [comment]
终于不是被骗才看到这个了
